llvm/tools/llvm-readobj/ELFDumper.cpp

```diff
@@ -5272,8 +5272,16 @@ static std::string getGNUProperty(uint32_t Type, uint32_t DataSize,
     return OS.str();
   case GNU_PROPERTY_AARCH64_FEATURE_1_AND:
   case GNU_PROPERTY_X86_FEATURE_1_AND:
-    OS << ((Type == GNU_PROPERTY_AARCH64_FEATURE_1_AND) ? "aarch64 feature: "
-                                                        : "x86 feature: ");
+
+    if (Type == GNU_PROPERTY_AARCH64_FEATURE_1_AND) {
```

Comment:
Add a static_assert that GNU_PROPERTY_AARCH64_FEATURE_1_AND and GNU_PROPERTY_RISCV_FEATURE_1_AND have the same value.